fury-on-vms/README.md

```diff
@@ -216,6 +216,17 @@ Next we need to define the masters node and the worker nodes. The FQDN that will
 
 For example, `master1` will become `master1.example.tld`.
 
+#### Custom registry for Kubernetes core components images
+
+We can override the URL of the registry where to pull images from for the Kubernetes core components (kube-apiserver, kube-controller-manager, kube-scheduler, kube-proxy, coredns). The host is mandatory, while the port is optional. Always append `/fury/on-premises` at the end (the default value is `registry.sighup.io/fury/on-premises`).
```

Comment:
Do you actually need to put the images under `/fury/on-premises` on your registry too? Can't you use a different path?

Reply:
You're right, the `/fury/on-premises` path is not always needed with a custom registry. It's necessary only when preserving the mirrored structure of the official registry. For others use cases a different path can be used. I propose updating the readme to reflect that.

Reply:
Go ahead, feel free to push the fix to this branch. I think the same applies to the registry for the kubernetes components